Composer/packages/server/src/constants.ts

```diff
@@ -20,3 +22,5 @@ export const APPINSIGHTS_INSTRUMENTATIONKEY = process.env.APPINSIGHTS_INSTRUMENT
 export const piiProperties = ['projectId', 'rootProjectId'];
 
 export const COMPOSER_VERSION = '1.3.1';
+
+export const templateSortOrder = [{ generatorName: '@microsoft/generator-microsoft-bot-empty', displayName: 'Blank bot' }, { generatorName: '@microsoft/generator-microsoft-bot-conversational-core', displayName: 'Basic conversational bot' }, { generatorName: QnABotTemplateId, displayName: 'QnAMaker bot' }]
```

Comment:
I think this (and the sorting func) could be extracted into a util and tested. It's not really a shared constant.

Reply:
Handled